### DISCUSSION TITLE
Reasoning time related issues

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1851] [General]
Hello, if I want to know when the recognition model infers each image and which script should be used for testing

**Comment by gaotongxiao** [comment]
Not quite sure what you're referring to, but here are some relevant docs:

https://mmocr.readthedocs.io/en/dev-1.x/user_guides/train_test.html

https://mmocr.readthedocs.io/en/dev-1.x/user_guides/inference.html

**Comment by htaoruan** [comment]
OK Thank you
